engine/src/main/java/io/camunda/zeebe/engine/state/instance/DbJobState.java

```diff
@@ -212,6 +213,12 @@ public JobRecord updateJobRetries(final long jobKey, final int retries) {
     if (job != null) {
       job.setRetries(retries);
       updateJobRecord(jobKey, job);
+
+      if (retries <= 0) {
+        makeJobNotActivatable(job.getTypeBuffer());
```

Comment:
:thinking: This part is not reachable. The [processor](https://github.com/camunda/zeebe/blob/main/engine/src/main/java/io/camunda/zeebe/engine/processing/job/JobUpdateRetriesProcessor.java#L38) doesn't accept if retries are less than 1.

We could remove this part.

Reply:
I have removed the changes to the `updateJobRetries` in its entirety, because of your comment below this.